.mailmap

```diff
@@ -0,0 +1,122 @@
+
+# Former Ambassador Labs employees (with a new address)
+Abhay Saxena <ark3@email.com>                          <ark3@datawire.io>
+Alice Nodelman <alice@nodelman.net>
+Alice Nodelman <alice@nodelman.net>                    <anode@boat.local>
+Alice Nodelman <alice@nodelman.net>                    <anode@datawire.io>
+Alvaro Saurin <alvaro.saurin@gmail.com>                <1841612+inercia@users.noreply.github.com>
+Alvaro Saurin <alvaro.saurin@gmail.com>                <alvaro@datawire.io>
+Ava Hahn <ava@aidanis.online>                          <87328374+aidanhahn@users.noreply.github.com>
+Ava Hahn <ava@aidanis.online>                          <aidanhahn@datawire.io>
+Cynthia Coan <cynthia@coan.dev>
+Cynthia Coan <cynthia@coan.dev>                        <SecurityInsanity@users.noreply.github.com>
+Donny Yung <donaldryung@gmail.com>                     <donaldyung@datawire.io>
+Flynn <emissary@flynn.kodachi.com>                     <flynn+github@kodachi.com>
+Flynn <emissary@flynn.kodachi.com>                     <flynn@datawire.io>
+Flynn <emissary@flynn.kodachi.com>                     <flynn@ryoohko.local>
+Flynn <emissary@flynn.kodachi.com>                     <kflynn@users.noreply.github.com>
+Gabriel Linden Sagula <gsagula@gmail.com>
+Itamar Turner-Trauring <itamar@itamarst.org>           <itamar@datawire.io>
+John Esmet <john.esmet@gmail.com>                      <jesmet@appnexus.com>
+John Esmet <john.esmet@gmail.com>                      <johnesmest@datawire.io>
+John Esmet <john.esmet@gmail.com>                      <johnesmet@datawire.io>
+Luke Shumaker <lukeshu@lukeshu.com>                    <luke@lukeshu.com>
+Luke Shumaker <lukeshu@lukeshu.com>                    <lukeshu@datawire.io>
+Noah Krause <krausenoah@gmail.com>
+Noah Krause <krausenoah@gmail.com>                     <nkrause@MacBook-Pro.local>
+Noah Krause <krausenoah@gmail.com>                     <nkrause@Opss-MacBook-Air.local>
+Noah Krause <krausenoah@gmail.com>                     <nkrause@datawire.io>
+Noah Krause <krausenoah@gmail.com>                     <noahkrause@Noahs-MacBook-Pro.local>
+Philip Lombardi <plombardi89@gmail.com>
+Philip Lombardi <plombardi89@gmail.com>                <893096+plombardi89@users.noreply.github.com>
+Philip Lombardi <plombardi89@gmail.com>                <plombardi@datawire.io>
+Shahriar Rostami <srdsecond@gmail.com>                 <54422255+alphashr@users.noreply.github.com>
+Stephanie Coyle <stephaniecoyle391@gmail.com>
+Stephanie Coyle <stephaniecoyle391@gmail.com>          <55929576+scoyle391@users.noreply.github.com>
+Stephanie Coyle <stephaniecoyle391@gmail.com>          <scoyle@datawire.io>
+
+# Outside contributors (inconsistent email -- did I choosed the correct email?)
+Brandon Catcho <bcatcho@doubledutch.me>                <bcatcho@gmail.com>
+Christoph Grossegger <christoph.grossegger@gmail.com>  <christoph@dust-bit-games.com>
+Dan Sipple <dsipple@etsy.com>                          <sipple.dan@gmail.com>
```

Comment:
Dan Sipple was at Datawire for awhile, actually, possibly as a contractor? Not sure it matters much.

Reply:
He was a contractor, and never had an `@datawire.io` email. The "employee/former-employee/non-employee" distinctions were for me, answering "should `@datawire.io` be canonical".

Reply:
I've now added a comment about that.